Source/WebCore/style/values/inline/StyleLineHeight.cpp

```diff
@@ -78,8 +78,9 @@ auto CSSValueConversion<LineHeight>::operator()(BuilderState& state, const CSSPr
     // values and raw numbers to percentages.
     if (primitiveValue.isPercentage()) {
         // FIXME: percentage should not be restricted to an integer here.
+        auto textZoom = shouldUseEvaluationTimeZoom(state) ? conversionData.zoom() : 1.0f;
         return LineHeight::Fixed {
-            CSS::clampToRange<LineHeight::Fixed::range, float>((state.style().computedFontSize() * primitiveValue.resolveAsPercentage<int>(conversionData)) / 100.0, minValueForCssLength, maxValueForCssLength)
+            CSS::clampToRange<LineHeight::Fixed::range, float>((state.style().fontDescription().computedSizeForRangeZoomOption(conversionData.rangeZoomOption()) * primitiveValue.resolveAsPercentage<int>(conversionData) * textZoom) / 100.0, minValueForCssLength, maxValueForCssLength)
```

Comment:
is there a specific reason we pass `conversionData.rangeZoomOption()` here? rather than using type's intrinsic zoom option? `LineHeight::Fixed::range.zoomOptions`